### PR TITLE
Announce OptaPlanner 9.34.0.Beta1

### PR DESCRIPTION
**Pull request by triceo**

### CODE REVIEW

File: optaplanner-website-root/content/blog/2023/02/15/OptaPlanner-9-is-coming.adoc [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+== When will OptaPlanner 9 be Final?
+
+At this point, we are aiming to release a Final version of OptaPlanner 9 shortly after Quarkus 3 releases theirs.
+All signs point to spring of 2023.
+
+
+== How can I help?
+
+We would appreciate if you could try to migrate your projects to the 9.35.0.Alpha3 version we've released to day.
```
Comment (rsynek) [inline optaplanner-website-root/content/blog/2023/02/15/OptaPlanner-9-is-coming.adoc:85]:
```suggestion
We would appreciate if you could try to migrate your projects to the 9.35.0.Alpha3 version we've released today.
```

**Review by rsynek — COMMENTED** [review]
LGTM; the exact releases may differ in the end and we might just refer to the Quarkus 3 OpenRewrite recipe.

File: optaplanner-website-root/content/blog/2023/02/21/OptaPlanner-9-is-coming.adoc [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+----
+rewrite:run \
+  -Drewrite.configLocation="${optaplanner_file}" \
+  -Drewrite.recipeArtifactCoordinates=org.optaplanner:optaplanner-migration:"$project_version" \
+  -Drewrite.exclusions=optaplanner-operator/**,optaplanner-examples/data/** \
+  -Drewrite.activeRecipes=org.optaplanner.openrewrite.Quarkus3 \
+  -Dfull \
+  -Dquickly \
+  -Dmigration
+----
```
Comment (triceo) [inline optaplanner-website-root/content/blog/2023/02/21/OptaPlanner-9-is-coming.adoc:92]: @rsynek Please provide the correct command.

File: optaplanner-website-root/content/blog/2023/02/21/OptaPlanner-9-is-coming.adoc [hunk 3]
```diff
@@ -0,0 +1,104 @@
+= OptaPlanner 9 is coming
+triceo
+2023-02-21
+:page-interpolate: true
+:jbake-type: post
+:jbake-tags: migration, jakarta, javax, javaee, quarkus, spring boot
+:jbake-social_media_share_image: migration-flowchart.png
+
+OptaPlanner 9 is on the way, and today we've made available its first Alpha release. But do not worry! Unless you use Quarkus or Spring Boot, absolutely nothing will change for you. Continue reading to find out what it is that we're doing, why, and how you can participate.
```
Comment (danielezonca) [inline optaplanner-website-root/content/blog/2023/02/21/OptaPlanner-9-is-coming.adoc:9]:
```suggestion
OptaPlanner 9 is on the way, and today we've made available its first Beta release. But do not worry! Unless you use Quarkus or Spring Boot, absolutely nothing will change for you. Continue reading to find out what it is that we're doing, why, and how you can participate.
```

File: optaplanner-website-root/content/blog/2023/02/21/OptaPlanner-9-is-coming.adoc [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+</dependency>
+----
+
+For users of `optaplanner-quarkus` or `optaplanner-spring-boot-starter`, the changes will be larger:
+
+* All imports of `javax.*` packages will need to be replaced by their `jakarta.*` equivalents, including changes to dependencies.
+* Dependencies on Quarkus 2 need to be changed to Quarkus 3.
+* Dependencies on Spring Boot 2 need to be changed to Spring Boot 3. In case of Spring Boot 3, a migration to Java 17 is also necessary. That is the decision of the Spring framework team, and we can not influence that.
+
+However, we have created tooling to simplify this transition to a point where it is also a single line of code. Simply run the following command while in the root directory of your project:
```
Comment (triceo) [inline optaplanner-website-root/content/blog/2023/02/21/OptaPlanner-9-is-coming.adoc:80]:
```suggestion
We have created tooling to simplify this transition to a point where it is also a single line of code. Simply run the following command while in the root directory of your project:
```

File: optaplanner-website-root/content/blog/2023/02/21/OptaPlanner-9-is-coming.adoc (hunk 2 above)
Comment (rsynek) [reply optaplanner-website-root/content/blog/2023/02/21/OptaPlanner-9-is-coming.adoc]: I think something along these lines could work after merging https://github.com/kiegroup/optaplanner/pull/2635 :

```
curl -o ToLatest9.yml https://raw.githubusercontent.com/kiegroup/optaplanner/main/optaplanner-migration/src/main/resources/META-INF/rewrite/ToLatest9.yml
mvn clean org.openrewrite.maven:rewrite-maven-plugin:LATEST:run \
   -Drewrite.configLocation=ToLatest9.yml \
   -Drewrite.recipeArtifactCoordinates=org.optaplanner:optaplanner-migration:8.34.0.Final \
   -Drewrite.activeRecipes=org.optaplanner.migration.ToLatest9
```

**Review by rsynek — APPROVED** [review]
All looks good, but let me suggest one addition: in most blog posts, we also provide the Summary section below. I think it's a good opportunity to repeat that:
- 8 and 9 are functionally equal
- there is an automated migration
which I think are the most important messages of this post.

**Review by rsynek — APPROVED** [review]

**Pull request merged**